service/controller/chart.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/giantswarm/chart-operator/service/controller/v1"
 )
 
+const chartControllerSuffix = "-chart"
```

Comment:
Note I added this constant, and add this suffix to controller name despite the fact there is only a single controller in this operator. I think it's a good idea to add those in the fist place to keep the convention.

Reply:
I missed that discussion. OK, let's then remove that when this change is delivered.

Reply:
I missed that discussion. But that means we'll have to provide both `Name` and `FinalizerName` to the `ControllerConfig`?

Reply:
Yes.